test/exercises_test.dart

```diff
@@ -45,11 +68,18 @@ Running tests for: $packageName
   await runCmd(["cp", "lib/${packageName}.dart", "lib/${packageName}.dart.bu"]);
   await runCmd(["cp", "test/${packageName}_test.dart", "test/${packageName}_test.dart.bu"]);
   try {
-    for (var cmds in [
-      ["cp", "lib/example.dart", "lib/${packageName}.dart"], // Replace main file with example
-      ["sed", "-i", "-e", "s/\\bskip:\\s*true\\b/skip: false/g", "test/${packageName}_test.dart"], // Enable all tests
-      ["pub", "get"], // Pull dependencies
-      ["pub", "run", "test"] // Run tests
+    for (dynamic cmds in [
```

Comment:
The `cmds` variable should be `List<String>` . 
Use `dynamic` as a function return type when the function returns a `string` or a `map` or `whatever data type`.
You can also call any function on a `dynamic` variable assuming it exists. 
```dart
(foo as dynamic).whatever(); //valid. compiler won't check if whatever() exists
(foo as var).whatever(); //illegal. var is not a type
```
Use `var` when for a variable you expect its data type to change during its lifetime.
e.g.

```dart
dynamic myFunc(int a){
  if (a == 1) return "String";
  if (a == 2) return [1,2,3]
  return null
}

/// Here it becomes a String.
var myVariableDataTypeVariable = myFunc(1);
/// Here it becomes a List.
var myVariableDataTypeVariable = myFunc(1);
/// Here it becomes null.
var myVariableDataTypeVariable = myFunc(1);

```

In conclusion previously `var` was fine. But we should explicitly make it of type `List<String>`.